frontend/packages/console-app/src/models/index.ts

```diff
@@ -1,23 +1,10 @@
 import { K8sKind } from '@console/internal/module/k8s';
 
 export const WorkspaceModel: K8sKind = {
-  kind: 'Workspace',
-  label: 'Workspace',
-  labelPlural: 'workspaces',
-  apiGroup: 'workspace.che.eclipse.org',
-  apiVersion: 'v1alpha1',
-  abbr: 'WS',
-  namespaced: true,
-  crd: true,
-  plural: 'workspaces',
-  propagationPolicy: 'Foreground',
-};
-
-export const DevWorkspaceModel: K8sKind = {
   kind: 'DevWorkspace',
-  label: 'Devworkspace',
-  labelPlural: 'workspaces',
-  apiGroup: 'apiextensions.k8s.io',
```

Comment:
This change seems strange: from `apiextensions.k8s.io` to `devfile.io`. Is it a CR or a CRD ?

Reply:
Hmm. Good point -- I'll look into it.

Reply:
I'm not sure why it was ever the CRD here -- we need to find a specific instance of a worksapce, so `devfile.io` is appropriate given https://github.com/che-incubator/che-workspace-operator/pull/92

Reply:
The frontend needs to create/edit DevWorkspace instances, so `devfile.io` is the correct option here. You may be thinking of the earlier functionality that checked if the CRD was defined on the cluster for enabling/disabling the feature -- this has been replaced by a back-end endpoint.

Reply:
Ah, OK